server/src/servershot.js

```diff
@@ -100,12 +100,23 @@ class Shot extends AbstractShot {
         // If the key doesn't already exist, go through the clips being inserted and
         // check to see if we need to store any data: url encoded images
         return this.convertAnyDataUrls(client, json, possibleClipsToInsert).then((oks) => {
+          let contentId = uuid.v4();
+          let head = json.head;
+          let body = json.body;
+          json.head = "";
+          json.body = "";
           return db.queryWithClient(
             client,
             `INSERT INTO data (id, deviceid, value)
              VALUES ($1, $2, $3)`,
             [this.id, this.ownerId, JSON.stringify(json)]
           ).then((rows) => {
```

Comment:
I guess there shouldn't be `rows` parameters here or below, since it is an insert